examples/cifar10_to_monai.ipynb

```diff
@@ -0,0 +1,606 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "GroundTruth:    cat  ship  ship plane\n",
+      "Predicted:    cat  ship  ship  ship\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataiter = iter(testloader)\n",
+    "images, labels = dataiter.next()\n",
+    "\n",
+    "# print images\n",
+    "imshow(torchvision.utils.make_grid(images))\n",
```

Comment:
Same as the preview logic. plt.subplot() should be better.

Reply:
Will use same resolution as above.

Reply:
OK, let's keep it.